### PR TITLE
Changes to have more generic project files + Other things...

### PR DESCRIPTION
**Pull request by Jj0YzL5nvJ**
https://github.com/mupen64plus/mupen64plus-core/pull/694

### CODE REVIEW

File: projects/msvc/mupen64plus-ui-console.sln [hunk 1]
```diff
@@ -32,14 +34,14 @@ Global
 		{0E29D9F8-C675-4D21-AF16-AA80EDDB264E}.Release|Win32.Build.0 = Release|Win32
 		{0E29D9F8-C675-4D21-AF16-AA80EDDB264E}.Release|x64.ActiveCfg = Release|x64
 		{0E29D9F8-C675-4D21-AF16-AA80EDDB264E}.Release|x64.Build.0 = Release|x64
-		{92D3FEB9-2129-41C5-8577-BCD7D961EF41}.Debug|Win32.ActiveCfg = Debug|Win32
-		{92D3FEB9-2129-41C5-8577-BCD7D961EF41}.Debug|Win32.Build.0 = Debug|Win32
-		{92D3FEB9-2129-41C5-8577-BCD7D961EF41}.Debug|x64.ActiveCfg = Debug|x64
-		{92D3FEB9-2129-41C5-8577-BCD7D961EF41}.Debug|x64.Build.0 = Debug|x64
+		{92D3FEB9-2129-41C5-8577-BCD7D961EF41}.Debug|Win32.ActiveCfg = New_Dynarec_Debug|Win32
```
Comment (Narann) [inline projects/msvc/mupen64plus-ui-console.sln]: Does New_Dynarec makes sense in the case of ui-console?

**Comment by Gillou68310** [comment]
Yeah the sln should not have a New_Dynarec config

**Comment by Gillou68310** [comment]
NVM

**Comment by Jj0YzL5nvJ** [comment]
It's done?

**Comment by Narann** [comment]
Can you rebase on master?

**Comment by Gillou68310** [comment]
And squash commit please?

**Comment by Jj0YzL5nvJ** [comment]
> And squash commit please?

With pleasure, fewer things to comment for me. I will do the same with all the others.

**Comment by Jj0YzL5nvJ** [comment]
P.S: Wrong encoding, now all project files are UTF-8 with BOM

**Pull request merged**